packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextLayoutManagerMapBuffer.java

```diff
@@ -378,6 +382,54 @@ private static Layout createLayout(
     return layout;
   }
 
+  public static void adjustSpannableFontToFit(
+    Spannable text,
+    float width,
+    YogaMeasureMode widthYogaMeasureMode,
+    float height,
+    YogaMeasureMode heightYogaMeasureMode,
+    double minimumFontSizeAttr,
+    double maximumFontSizeAttr,
```

Comment:
This is unused

Reply:
Removed in https://github.com/facebook/react-native/pull/44075/commits/367f65bad5a8521c02e1d63ab582d1d3c1668ecd